Update MS teams docs (#32395)
demisto/content: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

This integration was integrated and tested with version 1.0 of Microsoft Teams.

**Note:** 

> - This integration is supported in Cortex XSOAR 8 and up without using an engine.

Reviewer: ```suggestion
- This integration is supported in Cortex XSOAR 8 when not using an engine.
```

Author: @ShirleyDenkberg in this note, i'm saying that its supported in xsoar 8 with no need to an engine (but they can use an engine), so i guess "when not" is not the right expression

Reviewer: ok
